MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -1957,14 +1991,6 @@
 			name = "Instructions Card";
 			sourceTree = "<group>";
 		};
-		B46C42C32908AF1B006304B0 /* Preview */ = {
-			isa = PBXGroup;
-			children = (
-				B46C42C42908AF50006304B0 /* BannerDismissalViewControllerSnapshotTests.swift */,
```

Comment:
I moved this test to the `MapboxNavigationTests` target, it was previously added to the `MapboxCoreNavigationTests`